src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -576,6 +576,9 @@
   <data name="SetColorSchemeParentCommandName" xml:space="preserve">
     <value>Select color scheme...</value>
   </data>
+  <data name="TabSearchCommandKey" xml:space="preserve">
+    <value>Tab Search</value>

```

Comment:
Do we want to make this a tad bit more... interactive? Perhaps something like "Search for tab..." (kinda like "Set the tab color..." - it's not _technically_ a nested command, but it is a sequence of actions)

Reply:
oh yeah "Search for tab..." looks pretty decent 👍